benchmarks/text-benchmarks.cabal

```diff
@@ -15,7 +15,7 @@ category:            Text
 build-type:          Simple
 tested-with:    GHC==9.0.1,
                 GHC==8.10.4, GHC==8.8.4, GHC==8.6.5, GHC==8.4.4,
-                GHC==8.2.2, GHC==8.0.2, GHC==7.10.3
```

Comment:
See the bottom of https://github.com/haskell/text/pull/314#issue-588849330

This (temporarily) drops support of GHC 7.10 for benchmarks only. The `text` package itself still supports GHCs back to 7.0. 

As @chessai pointed below, it could be a good idea to drop support of obsolete versions of GHC, but this is, strictly speaking, a separate matter, and not a decision I'm implementing in this PR.